Src/Microsoft.ApplicationInsights.Web.sln

```diff
@@ -113,9 +113,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PerformanceCollector.NuGet"
 		{FBFCB985-BBFE-4954-A2EF-8124A42F56C7} = {FBFCB985-BBFE-4954-A2EF-8124A42F56C7}
 	EndProjectSection
 EndProject
-Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Perf.Shared", "PerformanceCollector\Shared\Perf.Shared.shproj", "{092A02B5-9ACC-48B5-B317-428F71257196}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Unit.Tests", "PerformanceCollector\Unit.Tests\Unit.Tests.csproj", "{F254D4FB-428D-408E-8251-39BCA7B4B5CE}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Perf.NetFull.Tests", "PerformanceCollector\Perf.NetFull.Tests\Perf.NetFull.Tests.csproj", "{F254D4FB-428D-408E-8251-39BCA7B4B5CE}"
```

Comment:
separate projects for tests as opposed to giving targetframeworks = net45;netstandard1.6 is better  - this way VisualStudio can run tests from both targets. Else VS only runs tests from the 1st target in the list.